app/assets/stylesheets/hyrax/_footer.scss

```diff
@@ -22,6 +22,11 @@ body {
   bottom: 0;
   height: $footer-height;
   width: 100%;
+
+  a,
+  &.navbar-inverse .navbar-link {
```

Comment:
How is this changing the header color?  All of this is within a `.site-footer` parent selector?